problem.md

```diff
@@ -0,0 +1 @@
+The Web had its origins in a vision of a new kind of media system -- one which completed the democratizing work of Gutenberg by giving ordinary people the power to create and publish their work to an unlimited audience. While parts of this vision have been realized, the web has ironically given rise to a dramatically more centralized and consolidated landscape of data storage and distribution -- completely unlike how libraries manage content. The recent rise of autocratic tendencies in various jurisdictions (Turkey, the Philippines, Russia, the United States) is a salutary reminder of the fragility of any centralized data storage system. In a centralized model, however, the problem of data authority is simple -- the central warehouse authorizes a piece of data simply by storing it. In the decentralized web, that problem is more difficult, as are the problems of locating data, versioning it, and storing it reliably. 
```

Comment:
I feel like this is part of the problem, but I don't see any of the challenges that gave rise to EDGI archiving work, challenges from community science, etc... that I think also inform why people are interested in DT?

Reply:
agree. This was pulled only from the Protocol Labs docs, plus one phrase to reference some of flyingzumwalt's newer language on how the internet is not like libraries into it. I do think this paragraph could be a good set up for moving into the need for community archiving because it ends on a line of critique of the centralized web and its model of authority.